.gitattributes

```diff
@@ -1,3 +1,2 @@
 vmlinux.h linguist-vendored
-openvswitch.h linguist-vendored
```

Comment:
Should we remove `scripts/gen-btf-headers.sh` then? or change it to use bpftool to generate the combined vmlinux.h?